script/index.coffee

```diff
@@ -475,10 +475,7 @@ api.wrap = (user, main=true) ->
         if user.stats.lvl < 100
           user.balance -= 2
         # Save off user's level, for calculating achievement eligibility later
-        if user.stats.lvl < 100
-          lvl = user.stats.lvl
-        else
-          lvl = 100
+        lvl = user.stats.lvl
```

Comment:
Because you're no longer capping this at 100, this bit isn't strictly necessary; you'd then use `user.stats.lvl` later instead of `lvl`.